app/views/checkout/payment.scala.html

```diff
@@ -140,23 +140,16 @@ <h2 class="fieldset__headline">
                             @checkout.errorMessage("Please enter a valid postal code")
                         </div>
                         <div class="form-field">
-                            @*
-                                TODO: No values on checkboxes? Is this used yet?
-                            *@
-                            <h5 class="checkbox-title">
-                                I'd like to receive information about offers and developments from Guardian News &amp; Media Limited via:
-                            </h5>
-                            <label class="checkbox">
-                                <span class="checkbox__input">
-                                    <input type="checkbox" class="js-input">
-                                </span>
-                                <span class="checkbox__label">Email</span>
-                            </label>
-                            <label class="checkbox">
-                                <span class="checkbox__input">
-                                    <input type="checkbox" class="js-input">
-                                </span>
-                                <span class="checkbox__label">Post</span>
+                            <label class="label checkbox" for="receive-gnm-marketing">
```

Comment:
Original comment has been outdated:

Can you use the same markup pattern used above for this?

```html
<label class="checkbox">
  @defining(form.data.get("personal.receiveGnmMarketing")) { gnmMarketingOpt =>
    <span class="checkbox__input">
      <input type="checkbox"
             name="personal.receiveGnmMarketing"
             @(if (gnmMarketingOpt.contains("true")) "checked" else "")
             value="@gnmMarketingOpt">
    </span>
  }
  <span class="checkbox__label">
    I'd like to be kept up to to date by email with offers and developments from Guardian News &amp; Media Limited
  </span>
</label>

```